src/pages/errorpages/NotFound.tsx

```diff
@@ -1,14 +1,19 @@
+import BlockIcon from '@mui/icons-material/Block';
 import { Box, Typography } from '@mui/material';
-import { useTranslation } from 'react-i18next';
+import { Trans } from 'react-i18next';
 
 const NotFound = () => {
-  const { t } = useTranslation();
-
   return (
-    <Box data-testid="404" sx={{ mt: '4rem' }}>
-      <Typography variant="h3" component="h1">
-        {t('feedback.error.404_page')}
-      </Typography>
+    <Box data-testid="404" sx={{ m: '4rem 2rem' }}>
+      <Trans
+        i18nKey="feedback.error.404_page"
+        components={{
+          span: <span style={{ display: 'flex', gap: '0.5rem' }} />,
```

Comment:
Mulig mindre x-margin og sentrert flex items hadde passet mobil bedre?

<img width="412" height="416" alt="bilde" src="https://github.com/user-attachments/assets/78aaee81-8fb5-4c9e-82f1-e607ebc5aaeb" />

vs
<img width="413" height="421" alt="bilde" src="https://github.com/user-attachments/assets/f05be698-1408-42f2-aff1-8df0b508d0a7" />


```suggestion
    <Box data-testid="404" sx={{ m: '4rem 0.5rem' }}>
      <Trans
        i18nKey="feedback.error.404_page"
        components={{
          span: <span style={{ display: 'flex', gap: '0.5rem', alignItems: 'center' }} />,
```

Reply:
Bruken av `<span>` får meg også til å tenke generelt på bruken av span/div vs Box fra MUI. Vi bruker stort sett `<Box sx={...}>` overalt i dag, men vet ikke om det egentlig har noen fordeler over å bare bruke `<div style={...}>` som du har gjort her (bare med span). Generelt tenker jeg native web er bedre en MUI, så kanskje skulle vi bevegd oss mer over til en sånn løsning.

Det vi får med Box er enklere tilgang til ting som er satt i theme da, så om vi hadde vært gode på å bruke theme spacing bl.a. så hadde det kanskje vært en fordel med Box her, men det er vi jo ikke 😅. Så kanskje vi skal prøve å bevege oss litt mer over til native div/span/section fremover, der vi ikke bruker noe spesielt fra theme 🤔

Reply:
Synes det blir lettere å lese når man får litt mer luft rundt teksten, så tror jeg lar det være som det er. 

Tenker at å gå for en "native first" tilnærming gir mening, og kun bruke MUI når det er behov (som f.eks theme).